forch/port_state_manager.py

```diff
@@ -155,6 +158,13 @@ def handle_device_placement(self, mac, device_placement, static=False):
 
         return self._handle_disconnected_device(device_placement)
 
+    def _resolve_auto_sequester_config(self, auto_sequestering):
+        if auto_sequestering == PortBehavior.AutoSequestering.true:
```

Comment:
with the appropriate type changes this could be simplified to:

auto_sequestering = self._default_auto_sequestering if auto_sequestering == unknown

or something like that.